Packs/Hackuity/Integrations/Hackuity/Hackuity_test.py

```diff
@@ -0,0 +1,48 @@
+import json
+from functools import partial
+from pathlib import Path
+from typing import Any
+
+import pytest
+from Hackuity import (
+    Client,
+    hackuity_search_findings_command,
+    hackuity_search_vulnerabilities_command,
+)
+
+TEST_DATA_PATH = Path("test_data")
+
+
+def util_get_json(directory, filename):
+    with (TEST_DATA_PATH / directory / filename).with_suffix(".json").open(
+        mode="rb"
+    ) as f:
+        return json.loads(f.read())
+
+
+@pytest.mark.parametrize(
+    "command, expected_filename",
+    [
+        pytest.param(
+            hackuity_search_findings_command,
+            "search_findings",
+            id="search_findings",
+        ),
+        pytest.param(
+            partial(hackuity_search_vulnerabilities_command, hy_global_only=False),
+            "search_vulnerabilities",
+            id="search_vulnerabilities",
+        ),
+    ],
```

Comment:
what about the other commands?

Reply:
The `hackuity-dashboard-data` and `hackuity-dashboard-widgets` commands are provided in this first version of the integration as best-effort.

The `hackuity-search-provider-vulnerabilities` and `hackuity-search-vulndb-vulnerabilities` commands are very similar so I thought it was enough to add a test for only one of them, but tell me if you want me to add a test for the other one.

Reply:
there should be a test for each command even it some are similar